bin/deploy

```diff
@@ -142,6 +142,19 @@ def deploy(client, cluster, service, repo, image, run_migrations = false)
   puts "#{service} deployment probably complete, but this tool is pretty rudimentary so check for yourself"
 end
 
+def notify_slack(env, image, stage)
+  slack_hook = ENV["SLACK_HOOK"]
+  if slack_hook
+    if stage=="start"
+      puts `curl -X POST --data-urlencode 'payload={"icon_emoji": ":ghost:", "text": "starting to deploy #{image} to #{env}"}' #{slack_hook}`
+    elsif stage=="end"
+      puts `curl -X POST --data-urlencode 'payload={"icon_emoji": ":sharkdance:", "text": "finished deploying #{image} to #{env}"}' #{slack_hook}`
+    end
+  end
+end 
+
+notify_slack(env, image, "start")
 deploy(client, env, RAILS_SERVICE, REPO, image, true)
 deploy(client, env, RESQUE_WORKER_SERVICE, REPO, image, false)
 deploy(client, env, RESQUE_SCHEDULER_SERVICE, REPO, image, false)
+notify_slack(env, image, "end")
```

Comment:
put a begin/rescue around this then do a notify of failure

Reply:
ok done